analytic_resource_plan_stock_picking/README.rst

```diff
@@ -0,0 +1,19 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==========================================
+Analytic Resource Planning - get resources
+==========================================
+Module features:
+    - prepare the stock for the project
+    - create Purchase request if there are not enough qty in stock
+
+
+Usage
+=====
+
+Create the resource plan lines and approve:
+
+* The system will fetch the products from stock is available
+* Purchase Request will be created when there's not enough qty in stock
```

Comment:
Can you create a contributors section and add 
* Jordi Ballester <jordi.ballester@eficent.com>
* Aaron Henriquez <ahenriquez@eficent.com>

and serpent?